go.mod

```diff
@@ -1,6 +1,6 @@
 module fybrik.io/openmetadata-connector
 
-go 1.17
+go 1.19
 
 require (
 	fybrik.io/fybrik v1.1.0
```

Comment:
should it be v1.2.0 ?

Reply:
you changed the go.mod, but I don't see changes in go.sum, did you run `go mod tidy` ?

Reply:
I ran `make check` but there were no changes. now there are changes because I upgraded Fybrik version.